packages/vehicle-lifecycle-network/test/vda.js

```diff
@@ -124,4 +124,32 @@ describe('Vehicle Lifecycle Network', function() {
                 });
         });
     });
-});
\ No newline at end of file
+
+    describe('ScrapAllVehiclesByColour', function() {
+        it('should select vehicles by colour and change vehicles status to SCRAPPED', function() {
+            /* vehicle with beige colour
+               and id 123456789 resides
+               in reposritory
+            */
+            var vehicleToScrap = 'Beige';
+            var assetRegistry;
+            var vehicleId = '123456789';
+
+            var scrapVehicle = factory.newTransaction(NS_D, 'ScrapAllVehiclesByColour');
+            scrapVehicle.colour =  vehicleToScrap;
```

Comment:
you could set the color here directly, which would make the test case code smaller

Reply:
Incorporated this suggestion